css/game.css

```diff
@@ -214,14 +246,22 @@ img.missile {
 }
 
 .start {
-    display: none;
+    position: absolute;
+    right: 34px;
+    bottom: 8px;
     background-color: #b21123;
     color: #ffffff;
+    font-size: 2.4rem;
     margin: 0 auto;
     border-radius: 100%;
     border: 4px solid #cccccc;
     width: 100px;
     height: 100px;
     line-height: 90px;
     text-align: center;
```

Comment:
can we have some padding on this button, the text looks a bit odd

Reply:
changed in a09d7d0